src/site/stages/build/process-cms-exports/transformers/README.md

```diff
@@ -2,14 +2,53 @@
 
 The files in this directory are owned by the following teams:
 
-| Team          |   File                                 
-| ----------    | ------------------------------        
-| Facilities    | node-health_care_local_facility.js 
-|               | node-health_care_local_health_service 
-|               | node-health_care_region_detail_page      
-|               | node-health_services_listing 
-|               | node-locations_listing 
+| Team          |   File
+| ----------    | ------------------------------
+| Facilities    | node-health_care_local_facility.js
+|               | node-health_care_local_health_service
+|               | node-health_care_region_detail_page
+|               | node-health_services_listing
+|               | node-locations_listing
 |               | node-nca_facility
 |               | node-vba_facility
-|               | node-vet_center 
+|               | node-vet_center
 |               | paragraph-health_care_local_facility_servi
+|               |
+| Public Web    | block_content-promo
+|               | node-basic_landing_page
+|               | node-checklist
+|               | node-faq_multiple_q_a
+|               | node-full_width_banner_alert
+|               | node-landing_page
+|               | node-media_list_images
+|               | node-media_list_videos
+|               | node-q_a
+|               | node-step_by_step
+|               | node-support_resources_detail_page
+|               | node-support_service
+|               | paragraph-alert_single
+|               | paragraph-audience_topics
+|               | paragraph-button
+|               | paragraph-checklist
+|               | paragraph-checklist_item
+|               | paragraph-contact_information
+|               | paragraph-email_contact
+|               | paragraph-link_teaser
+|               | paragraph-list_of_link_teasers
+|               | paragraph-lists_of_links
+|               | paragraph-media_list_images
+|               | paragraph-media_list_videos
+|               | paragraph-process
+|               | paragraph-q_a
+|               | paragraph-q_a_group
+|               | paragraph-react_widget
+|               | paragraph-rich_text_char_limit_1000
+|               | paragraph-step
+|               | paragraph-step_by_step
+|               | paragraph-table
+|               | paragraph-wysiwyg
+|               | taxonomy_term-administration
+|               | taxonomy_term-audience_beneficiaries
+|               | taxonomy_term-lc_categories
+|               | taxonomy_term-products
+|               | taxonomy_term-topics
```

Comment:
Thanks for updating! Maybe we don't need this now since we have codeowners?

Reply:
VSP asked for this 🤷‍♂️ I could see codeowners being a better source of truth though since this readme doesn't actually mean anything and is very likely to become out of date